clients/restclient/rest_client.go

```diff
@@ -43,3 +50,23 @@ func (c RestClient) PurgeConfiguration(org, space string) error {
 	}
 	return nil
 }
+
+func (c RestClient) getCsrfToken() error {
+	_, err := executeRestOperation(c.TokenFactory, func(token runtime.ClientAuthInfoWriter) (interface{}, error) {
+		return c.Client.Operations.GetCsrfToken(nil, token)
+	})
+	return err
```

Comment:
Check the err here if is nil. If yes -> return baseclient.NewClientError(err) as in this way the status code and the http response will be contained. If not -> return nil.

Reply:
Done